tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# pylint: disable=redefined-outer-name
+import tensorflow as tf
+import pytest
+
+
+@pytest.fixture(scope="session")
+def dataset():
+    mnist = tf.keras.datasets.mnist
+    (x_train, y_train), (x_test, y_test) = mnist.load_data()
+    x_train, x_test = x_train / 255.0, x_test / 255.0
+    yield (x_train, y_train), (x_test, y_test)
```

Comment:
Would it make sense to use `return`? I think `yield` would make sense if there is some clean-up code later.